test/parallel/test-repl-tab-complete.js

```diff
@@ -203,6 +203,86 @@ testMe.complete('toSt', common.mustCall(function(error, data) {
   assert.deepStrictEqual(data, [['toString'], 'toSt']);
 }));
 
+// tab completion Symbol
```

Comment:
Nit: Please always use a capital letter as first character of a comment and punctuate them.